### PR TITLE
Add build args

### PR DESCRIPTION
**Pull request by ypoplavs**
This PR...

## Changes

- added arguments for Dockerfile and Makefile build. Tested the changes in CI

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -26,4 +26,4 @@ jobs:
         go-version: 1.17
 
     - name: Test
-      run: make test
+      run: make test ANALYTICS_TRACKING_ID=${{ secrets.ANALYTICS_TRACKING_ID }} ANALYTICS_API_KEY=${{ secrets.ANALYTICS_API_KEY }}
```
Comment (kylehodgetts) [inline .github/workflows/build.yml:29]:
```suggestion
      run: make test TelemetryToken=${{ secrets.TELEMETRY_TOKEN }}
```

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -58,6 +58,7 @@ jobs:
           push: ${{ github.event_name != 'pull_request' }}
           tags: ${{ steps.meta-kusk-gateway.outputs.tags }}
           labels: ${{ steps.meta-kusk-gateway.outputs.labels }}
+          build-args: ANALYTICS_API_KEY=${{ secrets.ANALYTICS_API_KEY }},ANALYTICS_TRACKING_ID=${{ secrets.ANALYTICS_TRACKING_ID }}
```
Comment (kylehodgetts) [inline .github/workflows/release.yaml:61]:
```suggestion
          build-args: TelemetryToken=${{ secrets.TELEMETRY_TOKEN }}
```

File: .github/workflows/release.yaml [hunk 3]
```diff
@@ -78,6 +79,7 @@ jobs:
           push: ${{ github.event_name != 'pull_request' }}
           tags: ${{ steps.meta-kusk-gateway-agent.outputs.tags }}
           labels: ${{ steps.meta-kusk-gateway-agent.outputs.labels }}
+          build-args: ANALYTICS_API_KEY=${{ secrets.ANALYTICS_API_KEY }},ANALYTICS_TRACKING_ID=${{ secrets.ANALYTICS_TRACKING_ID }}
```
Comment (kylehodgetts) [inline .github/workflows/release.yaml:82]:
```suggestion
          build-args: TelemetryToken=${{ secrets.TELEMETRY_TOKEN }}
```

**Review by kylehodgetts — CHANGES_REQUESTED** [review]

**Review by jasmingacic — APPROVED** [review]

**Comment by jasmingacic** [comment]
Sorry for committing Kyle's suggestions but for a moment i thought it was my PR :D

**Review by kylehodgetts — APPROVED** [review]

**Pull request merged**